independent-publisher-connectors/HubSpot CMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,569 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Template ID.",
+            "x-ms-summary": "template_id",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      },
+      "put": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Update a template (V2)",
+        "description": "Updates a template. If not all the fields are included in the body, we will only update the included fields.",
+        "operationId": "TemplatesUpdate",
+        "parameters": [
+          {
+            "name": "template_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Template ID.",
+            "x-ms-summary": "template_id",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "source": {
+                  "title": "Source",
+                  "type": "string",
+                  "description": "source"
+                }
+              },
+              "required": [
+                "source"
+              ]
+            },
+            "description": "Body",
+            "x-ms-summary": "body",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://app.hubspot.com/oauth/authorize",
+      "tokenUrl": "https://api.hubapi.com/oauth/v1/token",
+      "scopes": {
+        "forms files hubdb transactional-email settings.users.write conversations.visitor_identification.tokens.create integration-sync tickets e-commerce files.ui_hidden.read crm.objects.marketing_events.read accounting crm.objects.marketing_events.write crm.schemas.custom.read crm.objects.custom.read crm.objects.feedback_submissions.read sales-email-read crm.objects.custom.write forms-uploaded-files settings.users.read crm.import contacts content reports social automation settings.billing.write actions timeline business-intelligence oauth": "forms files hubdb transactional-email settings.users.write conversations.visitor_identification.tokens.create integration-sync tickets e-commerce files.ui_hidden.read crm.objects.marketing_events.read accounting crm.objects.marketing_events.write crm.schemas.custom.read crm.objects.custom.read crm.objects.feedback_submissions.read sales-email-read crm.objects.custom.write forms-uploaded-files settings.users.read crm.import contacts content reports social automation settings.billing.write actions timeline business-intelligence oauth"

```

Comment:
@troystaylor Can you please make sure that the given scopes are valid. It looks like a string, instead of a string array.